packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -273,12 +314,28 @@ function CategoryHeader({ category }: { category: CategoryWithDapps }) {
   )
 }
 
+function FeaturedDapp({ dapp, onPressDapp }: DappProps) {
+  const onPress = () => onPressDapp(dapp)
+
+  return (
+    <Card style={styles.card} rounded={true} shadow={Shadow.Soft}>
+      <TouchableOpacity style={styles.pressableCard} onPress={onPress} testID="featuredDapp">
```

Comment:
Would be better to use our `<Touchable />` component to conform to the platform style.

Reply:
I keep forgetting about this... 🤦‍♂️